File: ruby/07_maps.rb
person = {
  "name" => "ulas",
  "age" => 20,
  "city" => "istanbul"
}

puts person["name"]

person["age"] = 21
person["job"] = "student"

person.each do |key, value|
  puts "#{key} -> #{value}"
end

File: go/07_maps.go
package main

import "fmt"

func main() {
	person := map[string]interface{}{
		"name": "ulas",
		"age":  20,
		"city": "istanbul",
	}

	fmt.Println(person["name"])
	fmt.Println(person["age"])

	person["age"] = 21
	person["job"] = "student"
	fmt.Println(person)

	for key, value := range person {
		fmt.Println(key, "->", value)
	}
}
